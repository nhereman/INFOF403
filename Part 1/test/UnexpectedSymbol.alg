co Unexpected Symbol co

begin
	a := 2 - 3
	b := $
	c := a + b
	print(c)
end