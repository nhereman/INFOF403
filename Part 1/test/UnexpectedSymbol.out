token: begin	lexical unit: BEG
token: a	lexical unit: VARNAME
token: :=	lexical unit: ASSIGN
token: 2	lexical unit: NUMBER
token: -	lexical unit: MINUS
token: 3	lexical unit: NUMBER
token: b	lexical unit: VARNAME
token: :=	lexical unit: ASSIGN
Parse Error : Pattern error at line 4 : "$"
